### PR TITLE
Add vim-toml

### PR DESCRIPTION
**Pull request by 2rs2ts**
# What

Adds https://github.com/cespare/vim-toml

# Why

I want syntax highlighting for toml

# Checklist

- [x] ~~Send RFC email to Braintree developers _if change may be impactful_. Please include a link to this PR so that discussion about the pros and cons of the change remains linked to the proposed changes. **Recent examples include**: addition of linter and completer, no longer removing end-of-line whitespace on save, changing to fzf for file lookup.~~ not necessary

### CODE REVIEW

**Review by creisor — APPROVED** [review]
LGTM 🐐

**Review by djquan — APPROVED** [review]

**Comment by 2rs2ts** [comment]
Thank you for the reviews! I don't have write access so I will have to wait for someone else to merge.

**Pull request merged**
